lib/tasks/scsb.rake

```diff
@@ -36,6 +36,23 @@ namespace :scsb do
     IndexFunctions.process_scsb_dumps([dump], ENV['SET_URL'])
   end
 
+  desc "Adds a local dump file to the database"
+  task add_local_dump_file: :environment do
+    abort "usage: FILE=full-file-path.xml.gz rake add_local_dump_file" unless ENV['FILE']
+    ev = Event.new(success: true)
```

Comment:
I think the events page will fail to load in the UI unless it has some start / end dates. I almost never use the UI in development, though; doesn't seem worth blocking over.